siriuspy/siriuspy/pwrsupply/csdev.py

```diff
@@ -840,60 +840,74 @@ def _get_ps_common_propty_database():
         'TimestampUpdate-Mon': {'type': 'float', 'value': 0,
                                 'prec': 7, 'unit': 'timestamp'},
         'CtrlMode-Mon': {'type': 'enum', 'enums': _et.INTERFACE,
-                         'value': Const.Interface.Remote},
+                         'value': Const.Interface.Remote,
+                         'unit': 'ctrlmode'},
         # Common Variables
         'PwrState-Sel': {'type': 'enum', 'enums': _et.PWRSTATE_SEL,
-                         'value': Const.PwrStateSel.Off},
+                         'value': Const.PwrStateSel.Off,
+                         'unit': 'pwrstate'},
         'PwrState-Sts': {'type': 'enum', 'enums': _et.PWRSTATE_STS,
-                         'value': Const.PwrStateSts.Off},
+                         'value': Const.PwrStateSts.Off,
+                         'unit': 'pwrstate'},
         'CtrlLoop-Sel': {'type': 'enum', 'enums': _et.CLOSE_OPEN,
-                         'value': Const.OpenLoop.Open},
+                         'value': Const.OpenLoop.Open,
+                         'unit': 'ctrlloop'},
         'CtrlLoop-Sts': {'type': 'enum', 'enums': _et.CLOSE_OPEN,
-                         'value': Const.OpenLoop.Open},
+                         'value': Const.OpenLoop.Open,
+                         'unit': 'ctrlloop'},
         'OpMode-Sel': {'type': 'enum', 'enums': _et.OPMODES,
-                       'value': Const.OpMode.SlowRef},
+                       'value': Const.OpMode.SlowRef,
+                       'unit': 'opmode'},
         'OpMode-Sts': {'type': 'enum', 'enums': _et.STATES,
-                       'value': Const.OpMode.SlowRef},
+                       'value': Const.OpMode.SlowRef,
+                       'unit': 'opmode'},
```

Comment:
In this particular case, Sel and Sts have int values with different meanings, maybe it would be better to make the unit different

Reply:
maybe `opmode_sel` e `opmode_sts`?

Reply:
I hesitated at first and decided to add only one unit. But since you both think it is better to separate units, I have done it.